### DISCUSSION TITLE
[Feature Request]: Implementing an Automated Knowledge Fetch Model

### DISCUSSION THREAD

**Discussion by H0llyW00dzZ** [ChatGPTNextWeb/NextChat#4126] [Ideas]
### Problem Description

The current AI model requires manual updates to incorporate the most recent knowledge. This process can be slow and inefficient, leading to outdated information for `System Prompt`.

### Solution Description

I propose that we explore the creation of an Automated Fetch Model for Knowledge. This potential feature could enable the AI model to update its knowledge base either periodically or in real time. By adopting this feature, we could ensure that the AI consistently has access to the most recent information, which could consequently enhance the accuracy and overall quality of its responses.

> [!NOTE]  
> This might be a feature to consider for future implementation, as there currently isn't a solution to fetch information for the model.

As an example, observe how the knowledge base of the Gemini-Pro model has been updated:

![Gemini-Pro Update](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/17626300/30c89566-85a1-414f-93c1-8232a0b6453f)

**Comment by nextchat-manager** [comment]
Please follow the [issue template](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/blob/main/.github/ISSUE_TEMPLATE/bug_report.md) to update title and description of your issue.
